pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -4,6 +4,8 @@ title: "Professional Development Workshop Survey"
 :ruby
   dont_cache
   enrollment_code = request.splat_path_info[1..-1]
+  redirect (request.scheme || 'http') + ':' + CDO.studio_url("/pd/workshop_survey/#{enrollment_code}")
```

Comment:
Weird. It works [here](https://github.com/code-dot-org/code-dot-org/blob/da6fe16bb199a9dce429ea69a06759fa6c672c22/pegasus/routes/pd_routes.rb#L3-L4). Maybe you need the 302 code as well?

Anyway it's fine this way too.